snippets/rust.json

```diff
@@ -416,6 +416,13 @@
         ],
         "description": "impl … { … }"
     },
+    "inline": {
+        "prefix": "inline",
+        "body": [
+            "#[inline]"
```

Comment:
I understand. Perhaps a `inline` snippet to declare an inline function that is similar to the `test` snippet?